src/libstd/collections/hash/map.rs

```diff
@@ -95,53 +95,61 @@ fn test_resize_policy() {
     }
 }
 
-// The main performance trick in this hashmap is called Robin Hood Hashing.
-// It gains its excellent performance from one essential operation:
+// The main performance trick in this hashmap is called Robin Hood linear
+// probing hashing. It gains its excellent performance from one essential
```

Comment:
I'm not sure if there is a technical term for this, but "Robin Hood linear probing hashing" sounds awkward. Maybe "Robin Hood Hashing with linear probing?"